src/main/java/igrad/storage/JsonSerializableCourseBook.java

```diff
@@ -62,7 +63,7 @@ public JsonSerializableCourseBook(ReadOnlyCourseBook source) {
      *
      * @throws IllegalValueException if there were any data constraints violated.
      */
-    public CourseBook toModelType() throws IllegalValueException {
+    public CourseBook toModelType() throws IllegalValueException, CommandException {
```

Comment:
why do you need to throw command exception for this?

Reply:
i don't think need though, when doing storage, i'll remove it